_episodes/01-conda.md

```diff
@@ -58,27 +59,28 @@ where the community can contribute conda installation packages.
 You can search Anaconda Cloud
 to find the command line entry needed to install the package. e.g:
 ~~~
-$ conda install -c https://conda.anaconda.org/scitools iris
+$ conda install -c conda-forge iris
```

Comment:
No need to write the URL for the channel with modern conda. I also remove the mentioned to the `scitools` channel b/c it is deprecated an its use may lead to a broken environment.

Reply:
Thanks!